Remotion/Web/Development.WebTesting.TestSite.Shared/MultiWindowTest/Main.aspx

```diff
@@ -41,7 +42,7 @@
             <p><remotion:WebButton ID="LoadWindowFunctionInFrame" Text="Load WindowFunction in Frame" runat="server"/></p>
             <p><remotion:WebButton ID="LoadMainAutoRefreshingFrameFunctionInFrame" Text="Load Main-auto-refreshing FrameFunction in Frame" runat="server"/></p>
             <p><remotion:WebButton ID="LoadWindowFunctionInNewWindow" Text="Load WindowFunction in new Window" runat="server"/></p>
-            <p><asp:HyperLink ID="NavigateAway" Text="Navigate Away" NavigateUrl="~/Default.aspx" runat="server"/></p>
+            <p><asp:HyperLink ID="NavigateAway" Text="Navigate Away" runat="server"/></p>
```

Comment:
via `<%=... %>` instead?
It's more a question of style/consistency then anything else.

Reply:
> Server tags cannot contain <% ... %> constructs.